pgdump-is-not-a-backup-tool.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: 'If pg_dump is not a backup tool, what is?'
+description: 'pg_dump is a utility for backing up a PostgreSQL database, not anymore'
```

Comment:
```suggestion
description: 'pg_dump is a utility for backing up a PostgreSQL database, but not anymore'
```

Reply:
For me this feels like its not reading correctly unless we add in a "but" or change the "is" to "was"

Reply:
I kind of like addition of "was". 

'pg_dump was a utility for backing up a PostgreSQL database, not anymore'